src/firefly/js/tables/ui/BasicTable.jsx

```diff
@@ -30,21 +28,18 @@ export class BasicTable extends React.Component {
             columnWidths: makeColWidth(props.columns, props.data, props.showUnits)
         };
 
-        this.onResize = this.onResize.bind(this);
+        this.onResize =  debounce((size) => {
+                if (size) {
+                    var widthPx = size.width;
+                    var heightPx = size.height;
+                    this.setState({widthPx, heightPx});
+                }
+            }, 100);
```

Comment:
I don't know if it matters but just noticed that you used half waiting time in the debounce function from the previous settings.